src/main/resources/com/google/api/codegen/metadatagen/java/grpc/grpc_package.snip

```diff
@@ -0,0 +1,181 @@
+@snippet generate(metadata)
+  buildscript {
+    repositories {
+      mavenCentral()
+    }
+    dependencies { classpath 'com.google.protobuf:protobuf-gradle-plugin:0.7.7' }
+  }
+
+  apply plugin: 'java'
+  apply plugin: 'maven'
+  apply plugin: 'signing'
+  apply plugin: 'com.google.protobuf'
+
+  description = 'GRPC library for {@metadata.identifier}'
+  group = "com.google.api.grpc"
+  version = "{@metadata.packageVersionBound.lower}"
+  sourceCompatibility = 1.6
+  targetCompatibility = 1.6
+
+  repositories {
+    mavenCentral()
+    mavenLocal()
+  }
+
+  dependencies {
+    compile "com.google.protobuf:protobuf-java:{@metadata.protoVersionBound.lower}"
```

Comment:
You are missing protoGenPackageDeps:

https://github.com/googleapis/packman/blob/master/templates/java/build-alt.gradle.mustache#L27